pkg/kubelet/kubelet.go

```diff
@@ -120,7 +120,7 @@ const (
 
 	// Generic PLEG relies on relisting for discovering container events.
 	// The period directly affects the response time of kubelet.
-	plegRelistPeriod = time.Second * 3
+	plegRelistPeriod = time.Second * 1
```

Comment:
Just curious, what guides us for a proper relisting frequency ?
 Add a line note